mason/site/header/head.mas

```diff
@@ -18,6 +18,9 @@
   <!-- <link rel="stylesheet" type="text/css" href="/css/dataTables.bootstrap.min.css"> -->
   <link rel="stylesheet" type="text/css" href="/css/buttons/buttons.bootstrap.min.css">
   <link rel="stylesheet" type="text/css" href="/css/bootstrap3-xl-class.css">
+  <link rel="stylesheet" type="text/css" href="/css/leaflet.css">
```

Comment:
we can load these css files specifically on the locations page like https://github.com/solgenomics/sgn/blob/master/lib/SGN/Controller/BreedersToolbox/TrialComparison.pm#L19

Reply:
Got it, that does make more sense.